newdle/migrations/versions/20250701_c1_set_interrogation_participant_names.py

```diff
@@ -0,0 +1,28 @@
+"""Set interrrogation as default name for participants with no name
+
+Revision ID: c1
+Revises: 30b5fe7b5949
+Create Date: 2025-07-01 12:14
+"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'c1'
```

Comment:
Small nit, but the revision id should be generated by alembic. It seems like we don't have a script for this like we do in Indico, but you can run `alembic -c newdle/migrations/alembic.ini revision` which generates a new file and you can take the ID from there (also make sure to update this file's name). 

If it complains about missing script location (not sure why that is), you can add `script_location = newdle/migrations` to `alembic.ini`